.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: Run tests
+
+on: push
```

Comment:
I don't know enough about CircleCI to know what events triggered the workflow, but this LGTM